### DISCUSSION TITLE
Max length filename of uploaded files is limit at 50 character!

### DISCUSSION THREAD

**Discussion by Loclhh** [oqtane/oqtane.framework#1826] [General] [answered]
Max length filename of uploaded files is limited at 50 characters!
Therefore if you upload a file with length of name is more than 50 character then it will cause error!

Temporary solution: Change the size of field: name in table Files to 255 character (this based on max length of filename in MAC and Linux)

Hope this help for someone like me!

Nice day all!

**Comment by leigh-pointer** [comment, marked as answer]
@Loclhh I have already added this to the issue list #1788 There are basically two options, one; extend the field length and report any errors gracefully  when length is to long , or report the errors gracefully when length is to long .

**Reply by Loclhh** [reply]
Thank you!

**Comment by sbwalker** [comment]
Fixed in #1830
